### DISCUSSION TITLE
[v8] subrows and global filter

### DISCUSSION THREAD

**Discussion by brunoti** [TanStack/table#3828] [Q&A] [answered]
Is there a way to make sub rows filterable from the global filter? Now it ignores and kinda turns off the expansion plugin when searching. I think a good way would be to have a config option to flatten everything. Maybe?

**Comment by tannerlinsley** [comment, marked as answer]
I think you want to use the `filterFromLeafRows: true` option, which will filter leaf rows first, then parents instead of a top-down filter which is the default.

**Reply by kasunnirmala** [reply]
Still not able to work on the filter by sub rows, though this flag is set to true.
(Same as with the both column filtering, and global filtering)

![tanstack-table-example-expanding - CodeSandbox (1)](https://user-images.githubusercontent.com/44490878/189494053-bb350138-c974-4173-b904-5ec67ba6f5b2.gif)

**Reply by sumitaney** [reply]
@tannerlinsley What’s going on? 
Any update on this!!

**Reply by tannerlinsley** [reply]
Link me to a CodeSandBox and I’ll take a look.

**Reply by nomttr** [reply]
Is `filterFromLeafRows` available in the v7?

**Reply by Fraley96michael** [reply]
What was the answer, I'm not having this work either

**Reply by Ani-pejakala** [reply]
It works fine. You need pass filterFromLeafRows:true on table instance. Not columnDef. Thanks
